src/js/entities-service/entities/clinicians.js

```diff
@@ -120,6 +120,14 @@ const Collection = BaseCollection.extend({
   url: '/api/clinicians',
   model: Model,
   comparator: 'name',
+  filterAssignable() {
```

Comment:
@nmajor25 I'm running into a place where we have a similar pattern of wanting to filter a collection, but this pattern is mutating.  Now I think we should make it not mutating like:
```js
const assignable = ...;
return this.clone().reset(assignable);
```

Then the uses would need to change to:
```js
return clinicians.filterAssignable();
```

I don't think we need to update the release, but we should update this branch.

Reply:
Nope, sorry the `reset` returns an array of the new models.

So this has to be:
```js
const clone = this.clone();
clone.reset(assignable);
return clone;
```